app/controllers/volunteer_ops_controller.rb

```diff
@@ -5,7 +5,19 @@ class VolunteerOpsController < ApplicationController
   def index
     @volunteer_ops = VolunteerOp.order_by_most_recent
     @organisations = Organisation.where(id: @volunteer_ops.select(:organisation_id))
-    @markers = build_map_markers(@organisations)
+    harrow_markers = build_map_markers(@organisations)
+    doit_markers = '[]'
+    # Do-it API works from below
+    if Feature.active? :doit_volunteer_opportunities
+      host = "https://api.do-it.org"
+      href = "/v1/opportunities\?lat\=51.5978\&lng\=-0.3370\&miles\=0.5 "
+      @doit_orgs = Array.new
```

Comment:
Style/EmptyLiteral: Use array literal `[]` instead of `Array.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)